app/modules/intelligence/provider/provider_service.py

```diff
@@ -83,7 +86,12 @@ def get_large_llm(self):
             else "openai"
         )
 
-        if preferred_provider == "openai":
+        if self.environment != "development":
```

Comment:
I do not understand the motive and replacement of this action. what happens if we disable this llm?

Reply:
I have added this check as the first check during setting the development mode. Here we are skipping LLM initialization if Development mode is not enabled. The statement I have used here is wrong. I should be checking `if os.getenv("isDevelopmentMode") == enabled`. SHall update this